tests/validation/test_schemas.py

```diff
@@ -25,7 +25,7 @@ class GKSSchemaMapping(BaseModel):
     base_classes: set = set()
     concrete_classes: set = set()
     primitives: set = set()
-    schema: dict = dict()
+    schema: dict = {}
```

Comment:
as a side note, I tried to properly add ruff to vrs-python a few months ago, but the import sort ended up introducing a circular dependency. If it's not doing that anymore, then we could probably get it added in.